pkg/logs/decoder/line_handler.go

```diff
@@ -101,6 +103,8 @@ func (h *SingleLineHandler) process(message *Message) {
 // MultiLineHandler makes sure that multiple lines from a same content
 // are properly put together.
 type MultiLineHandler struct {
+	// Put expvar Int first because it's modified with sync/atomic, so it needs to
```

Comment:
Shouldn't this comment be moved to `type CountInfo`, as there is no `expvar` in this struct and `countInfo` is not the first field of the struct.

Reply:
oops this is an artifact I forgot to remove.